Configurations/10-main.conf

```diff
@@ -934,28 +934,28 @@ sub vms_info {
     # in order to build "universal" binary and allow OpenSSL take
     # advantage of NEON when it's available.
     #
-    "android-armeabi" => {
-        inherit_from     => [ "android", asm("armv4_asm") ],
+    "android-armeabi-base" => {
+        inherit_from     => [ asm("armv4_asm") ],
```

Comment:
This are practically indistinguishable from assemby module templates. One can argue that there should be common ELF-specific templates that can be used in both Linux and Android targets. Meanwhile I'd suggest to omit them and inherit directly from *_asm in compiler-specific targets. Yes, there will be duplicates, but as already said, it makes more sense to unify all ELF targets, which would be separate effort.